txtorcon/torcontrolprotocol.py

```diff
@@ -103,6 +104,25 @@ def unlisten(self, cb):
 
     def got_update(self, data):
         # print self.name,"got_update:",data
+        if self.use_stem:
+            import stem.response
+            response_data = ""
+            # Add appropriate headers for parsing into stem.response.ControlMessage instance
+            if self.name == "CIRC":
+                response_data = "650 " + self.name + " " + data.replace('\n', ' ') + "\r\n"
+            elif self.name == "NEWCONSENSUS" or self.name == "NS":
+                lines = data.split('\n')
```

Comment:
why do these events need to be special-cased? that is, what's special about CIRC/NEWCONSENSUS/NS?